set.mm

```diff
@@ -424500,165 +424615,138 @@ vertex and a (simple) path of length 2 from the other vertex back to the
     $( The number of simple paths of length 2 is n*(n-1) in a friendship graph
        with n vertices.  This corresponds to the proof of claim 3 in [Huneke]
        p. 2:  "... the paths of length two in G: by assumption there are ( n
-       2 ) such paths.".  However, the order of vertices is not respected by
-       Huneke, so he only counts half of the paths which are existing when
-       respecting the order as it is the case for simple paths represented by
-       words.  (Contributed by Alexander van der Vekens, 6-Mar-2018.)  (Revised
-       by AV, 16-May-2021.) $)
-    frgrhash2wsp $p |- ( ( G e. FriendGraph /\ V e. Fin /\ V =/= (/) )
+       2 ) such paths.".  However, the direction in which the vertices are
+       arranged is not respected by Huneke, so he only counts half of the paths
+       ( ` ( n _C 2 ) = ( ( n x. ( n - 1 ) ) / 2 ) ` ) which are existing if
+       the different arrangement is respected as it is the case for simple
+       paths represented by words.  (Contributed by Alexander van der Vekens,
```

Comment:
Do you mean: "Huneke counts undirected paths, so obtains the result ` ( n _C 2 ) = ( ( n x. ( n - 1 ) ) / 2 ) `, whereas we count directed paths, obtaining twice that number." ?

Reply:
You are right, my comment is a little bit clumsy/confusing - I have revised it accordingly.